user_guide_src/source/outgoing/response.rst

```diff
@@ -249,6 +251,27 @@ life, and is most secure when generated on the fly. To make this simple, you can
         . . .
     </style>
 
+.. warning:: If an attacker injects string like ``<script {csp-script-nonce}>``, it might become the real nonce attribute with this functionality. You can customize the placeholder string with the ``$scriptNonceTag`` and ``$styleNonceTag`` properties in **app/Config/ContentSecurityPolicy.php**.
```

Comment:
```suggestion
.. warning:: If an attacker injects a string like ``<script {csp-script-nonce}>``, it might become the real nonce attribute with this functionality. You can customize the placeholder string with the ``$scriptNonceTag`` and ``$styleNonceTag`` properties in **app/Config/ContentSecurityPolicy.php**.
```